portal/auth.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+from . import db
+
+from portal.db import get_db
+from flask import (
+    Flask, Blueprint, flash, g, redirect, render_template, request, url_for, session
+)
+from werkzeug.security import check_password_hash, generate_password_hash
+
+bp = Blueprint('auth', __name__)
+
+
+@bp.route('/', methods=('GET', 'POST'))
+def index():
+    if request.method == 'POST':
+        email = request.form['email']
+        password = request.form['password']
+        print(generate_password_hash(password))
```

Comment:
Lots of print statements left in production code. Fine for dev and testing, but should be removed before committing and submitting a PR for this exact reason. This line is particularly dangerous because you're adding all users' hashed passwords to the production logs.